php/libraries/Module.class.inc

```diff
@@ -161,4 +165,25 @@ class Module
         }
         return "No help available for $page ($helpFile)";
     }
+
+    /**
+     * Returns false if user does not have access to this module.
+     *
+     * Usage: the hasAccess() function in a child class should contain
+     * something like:
+     *
+     *     // create user object
+     *     $user =& User::singleton();
+     *
+     *     return $user->hasPermission('superuser');
+     *
+     * You do not need to overload hasAccess() if there are no access restrictions
+     *
+     * @return bool
+     * @access private
+     */
+    private function _hasAccess()
```

Comment:
shouldn't this be protected instead of private? What's the point of a private method that just does return true?